packages/zoe/src/contractFacet/types.js

```diff
@@ -202,7 +202,7 @@
  */
 
 /**
- * @typedef {{ zcfSeat: ZCFSeat, userSeat: ERef<UserSeat>}} ZcfSeatKit
+ * @typedef {{ zcfSeat: ZCFSeat, userSeat: ERef<UserSeat>}} ZCFSeatKit
```

Comment:
Looking at this from the perspective of generating release notes from conventional commits, I'm not sure this is a fix; i.e. I don't think this has an observable behavior change. Unless... is this type exported from the package? Then it might be seen as an API change for clients that depend on types.

Reply:
Yes, it's exported, but all the sdk users of it are indirect.

I'll update this to a "refactor".

Reply:
Actually, I will change this in the opposite direction (preserver the name of this type, but fix the incorrect reference) to make it a proper refactor and prevent the technically-breaking change.